lib/platform/bitbucket/index.md

```diff
@@ -3,8 +3,11 @@
 ## Unsupported platform features/concepts
 
 - Adding assignees to PRs not supported (does not seem to be a Bitbucket concept)
+- `automergeStrategy=rebase` [is unsupported][bcloud-16610] by Bitbucket Cloud.
```

Comment:
```suggestion
- `automergeStrategy=rebase` is unsupported [bcloud-16610] by Bitbucket Cloud.
```
Typo?

Reply:
This was a deliberate choice and not a typo, such that the text `is unsupported` is a link itself.

The initial commit displays like:
![image](https://user-images.githubusercontent.com/1691627/123562712-18602900-d776-11eb-9986-96b9686273fd.png)

The suggested change displays like:
![image](https://user-images.githubusercontent.com/1691627/123562727-2ca42600-d776-11eb-860d-ae91b328785a.png)

Which is more confusing to read in my opinion — it's not really a sentence. If the style of documents here is to make hyperlinks overt and explicit instead of hyperlinks in existing text, I can change this into something like the following?

```
- `automergeStrategy=rebase` is unsupported by Bitbucket Cloud (See: [BCLOUD-16610][bcloud-16610])
```

In light of this, would you prefer the intended commit, the suggested change, or the third option presented above?

Reply:
```suggestion
- `automergeStrategy=rebase` not supported by BitBucket Cloud, see [Jira issue BCLOUD-16610](https://jira.atlassian.com/browse/BCLOUD-16610)
```

It should always be clear what you'll get when you click on a link. So, I'll take: "modified version of third option". See my suggestion in this comment. 😉 

I don't know if that `[BCLOUD-16610][bcloud-16610])` link style will work on our final page, so I'm taking the precaution to use a fully qualified URL here. 😄